xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-macro/xwiki-platform-notifications-macro-ui/src/main/resources/XWiki/Notifications/Code/Macro/NotificationsMacro.xml

```diff
@@ -422,8 +422,11 @@ require(['jquery', 'XWikiAsyncNotificationsMacro', 'xwiki-events-bridge'], funct
       if (!entry.read) {
         // Style the read button
         readButton.addClass('notification-event-read-button').addClass('btn btn-xs');
-        // Insert the cross icon
-        readButton.html('$services.icon.renderHTML("check")');
+        // Insert the cross icon and an accessible label
+        var readButtonIcon = $('$services.icon.renderHTML("check")');
```

Comment:
Sounds like the quotes should be used in the other way: simple quote for velocity to not interprate `check` since it's a constant, and double quotes in javascript. 
```suggestion
        var readButtonIcon = $("$services.icon.renderHTML('check')");
```
You should fix the same way below.

Reply:
Note that Stephane has kept the previous behavior so we have to wonder whether it was working or not before his change.